app/views/home/_header.html.erb

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-expand-lg bg-body-tertiary">
+  <div class="container ">
+    <%= link_to "Recipe Rails", root_path, class:"navbar-brand"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+         <%= link_to "Foods", foods_path, class:"nav-link active"%>
+          <%# <a class="nav-link active" aria-current="page" href="#">Home</a> %>
+        </li>
+        <li class="nav-item">
+        <%= link_to "Recipes", recipes_path, class:"nav-link active"%>
+          <%# <a class="nav-link" href="#">Features</a> %>
+        </li>
+         <li class="nav-item">
+        <%= link_to "Public Recipes", public_recipes_path, class:"nav-link active"%>
+          <%# <a class="nav-link" href="#">Features</a> %>
```

Comment:
- [ ] Please kindly delete the commented lines of code to keep your code more clean 🙃